public/app/plugins/panel/table/models.gen.ts

```diff
@@ -1,21 +1,16 @@
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-// NOTE: This file will be auto generated from models.cue
-// It is currenty hand written but will serve as the target for cuetsy
+// This file was autogenerated by cuetsy. DO NOT EDIT!
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-import { TableCellDisplayMode, TableSortByFieldState } from '@grafana/ui';
+import * as ui from '@grafana/schema';
```

Comment:
Hmm, this change looks

Reply:
i'll leave to @ryantxu to explain the thinking about the shift from `@grafana/ui` to `@grafana/schema` itself. But we've shifted to wildcard imports because it's _vastly_ easier for codegen, and the codegen enforces that imports are _only_ allowed from that package (with the possibility of whitelisting others, later). Such restrictions are necessary to make scuemata guarantees about backwards compatibility machine-verifiable.